packages/keystone/src/lib/core/mutations/access-control.ts

```diff
@@ -13,15 +13,16 @@ import {
   PrismaFilter,
   resolveUniqueWhereInput,
   resolveWhereInput,
+  UniquePrismaFilter,
 } from '../where-inputs';
 
 export async function getAccessControlledItemForDelete(
   list: InitialisedList,
   context: KeystoneContext,
-  filter: UniqueInputFilter,
-  inputFilter: UniqueInputFilter
+  uniqueInput: UniqueInputFilter
 ): Promise<ItemRootValue> {
-  const itemId = await getStringifiedItemIdFromUniqueWhereInput(filter, list.listKey, context);
+  const uniqueWhere = await resolveUniqueWhereInput(uniqueInput, list.fields, context);
+  const itemId = await getStringifiedItemIdFromUniqueWhereInput(uniqueWhere, list.listKey, context);
```

Comment:
This change isn't right. `getStringifiedItemIdFromUniqueWhereInput` getting the value directly from GraphQL input is intentional because e.g. it will be in the string version of the id which `getStringifiedItemIdFromUniqueWhereInput` needs to be return.